build-compiler-rt.sh

```diff
@@ -123,8 +121,8 @@ for arch in $ARCHS; do
         -DCMAKE_C_FLAGS_INIT="$CFGUARD_CFLAGS" \
         -DCMAKE_CXX_FLAGS_INIT="$CFGUARD_CFLAGS" \
         $SRC_DIR
-    $BUILDCMD ${CORES+-j$CORES}
-    $BUILDCMD install
+    cmake --build . ${CORES:+-j${CORES}}
```

Comment:
There's a faint difference here in how expansion works between the previous `${CORES+...}` and `${CORES:+...}`. I believe your change is good though (we probably don't want to expand to `-j` if `$CORES` is set but is an empty string) - can you split that out into a separate commit before the rest of the refactoring?

Reply:
Done